core/src/main/scala/chisel3/Printf.scala

```diff
@@ -3,11 +3,13 @@
 package chisel3
 
 import scala.language.experimental.macros
-
 import chisel3.internal._
 import chisel3.internal.Builder.pushCommand
-import chisel3.internal.firrtl._
 import chisel3.internal.sourceinfo.SourceInfo
+import chisel3.experimental.BaseSim
+
+/** Named class for [[printf]]s. */
+final class PrintfId(val pable: Printable) extends BaseSim
```

Comment:
Anyone have a better name suggestion here? Didn't want to call it `Printf` and have it collide with the IR node.

Reply:
`NamedPrintf`?

Reply:
A collision with the IR node is fine, no users should ever run into that and we can easily distinguish internally